app/models/tasks/assign_tubes_to_wells_handler.rb

```diff
@@ -50,22 +50,25 @@ def do_assign_pick_volume_task(_task, params)
   # rubocop:todo Metrics/AbcSize
   def wells_with_duplicates(params) # rubocop:todo Metrics/CyclomaticComplexity
```

Comment:
If the CyclomaticComplexity todo is scoped to the method by declaring it this way, maybe the other three above should be too since they're turned back on after the method.